src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/wam/WamPageObject.java

```diff
@@ -180,40 +186,53 @@ public void clickNextPaginator() {
   }
 
   public void selectTab(WamTab tab) {
-    scrollAndClick(driver.findElement(By.cssSelector(String.format(WAM_TAB_CSS_SELECTOR_FORMAT,
-                                                                   tab.getId()
+    scrollAndClick(driver.findElement(By.cssSelector(String.format(
+        WAM_TAB_CSS_SELECTOR_FORMAT,
+        tab.getId()
     ))));
     isLoaded();
     verifyTabSelected(tab);
   }
 
   private void verifyTabSelected(WamTab tab) {
-    Assertion.assertTrue(driver.findElement(By.cssSelector(String.format(WAM_TAB_CSS_SELECTOR_FORMAT,
-                                                                         tab.getId()
+    Assertion.assertTrue(driver.findElement(By.cssSelector(String.format(
+        WAM_TAB_CSS_SELECTOR_FORMAT,
+        tab.getId()
     )))
                              .getAttribute("class")
                              .contains("icon-vertical-selected"));
     wait.forElementVisible(tabSelected);
   }
 
-  public void verifyLatestDateInDatePicker() {
+  /**
+   * Checks if yesterday's, day before yesterday, or current day is selected by default in date picker
+   * 3 days are checked due to lags in bulkimporter (providing data for WAM API)
+   */
+  public void verifyDateInDatePicker() {
     String currentDate = datePickerInput.getAttribute("value");
+    String dayBeforeYesterday = DateTimeFormat.forPattern("MMMM d, yyyy")
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "MMMM d, yyyy" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)